src/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/YearCreated.tsx

```diff
@@ -74,6 +75,14 @@ export const YearCreated: React.FC = () => {
           selected={`${createdBeforeYear}`}
         />
       </Flex>
+      <Checkbox
+        selected={allowEmptyCreatedDates}
+        onSelect={(allowEmpty: boolean) => {
+          filterContext.setFilter("allowEmptyCreatedDates", allowEmpty)
+        }}
+      >
```

Comment:
Looks great! My only comment is it seems like the checkbox is a little too close to the select. Maybe try adding a margin-top by using the `mt` prop. `mt` takes a [space unit](https://palette.artsy.net/tokens/spacing/) which is a multiple of 10px. Maybe a 5px buffer here (or `0.5` units) would be good.

Reply:
Awesome thanks for the help here!

Just pushed an update -- looking better:
<img width="290" alt="Screen Shot 2020-04-15 at 9 45 45 AM" src="https://user-images.githubusercontent.com/12748344/79344923-72bcf300-7efe-11ea-8af7-95624310490c.png">

Reply:
Yeah! Looks great! 🚢